crates/forge/src/result.rs

```diff
@@ -244,7 +245,9 @@ impl TestSetup {
                 // force the tracekind to be setup so a trace is shown.
                 traces.extend(err.traces.map(|traces| (TraceKind::Setup, traces)));
                 logs.extend(err.logs);
-                labeled_addresses.extend(err.labels);
+                labeled_addresses.extend(
+                    err.labels.into_iter().map(|l| (b160_to_h160(l.0), l.1)).collect::<Vec<_>>(),
```

Comment:
no need to collect

Reply:
done in https://github.com/foundry-rs/foundry/pull/5768/commits/3e18639d0ca47edd9c4d8aed36e8c14300eed8fa